frontend/src/containers/pages/PlayScenarioPage/PlayScenarioResolver.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useContext, useState } from "react";
+import {
+  Route,
+  Switch,
+  useHistory,
+  useLocation,
+  useParams,
+} from "react-router-dom";
+import axios from "axios";
+
+import AuthenticationContext from "context/AuthenticationContext";
+import useGraph from "hooks/useGraph";
+
+import PlayScenarioPage from "./PlayScenarioPage";
+import PlayScenarioPageMulti from "./PlayScenarioPageMulti";
+
+// TODO: move this somewhere else and add error handling
+async function get(url, userIdToken) {
```

Comment:
@JordanBlenn We should make a ticket to make sure we don't forget about this :)

Reply:
Added a placeholder ticket to the backlog but I'll need you to fill out the details if that's all good :)